ios/Classes/Module/FacebookModule.m

```diff
@@ -16,6 +16,7 @@
 #import "TiBase.h"
 #import "TiHost.h"
 #import "TiUtils.h"
+#import "TiBlob.h"
```

Comment:
I'm curious why this does not cause issues for us right now. Maybe it works if the symbol is linked by other modules? Good catch in any case!